boards/arm/sama5/sama5d4-ek/Kconfig

```diff
@@ -323,8 +323,7 @@ endif # SAMA5D4EK_ROMFS_MOUNT
 
 config SAMA5D4EK_USBHOST_STACKSIZE
 	int "USB host waiter stack size"
-	default 1536 if USBHOST_HUB
-	default 1024 if !USBHOST_HUB
```

Comment:
It is inappropriate to increase the size of this tuned stack size.  Let's not do that.